quantus/metrics/complexity/sparseness.py

```diff
@@ -273,3 +259,30 @@ def evaluate_instance(
             a.shape[0] * np.sum(a)
         )
         return score
+
+    def evaluate_batch(
+        self, *, x_batch: np.ndarray, a_batch: np.ndarray, **_
+    ) -> List[float]:
+        """
+        TODO: write meaningful docstring about what does it compute.
```

Comment:
@annahedstroem I would like to ask for your help with `TODO: write meaningful docstring about what does it compute.`. I wrote the explanation for those I know or could understand, but I lack the knowledge about the rest. Of course, I can remove it, and put generic `Implementation of evaluate batch`, but I doubt it would be helpful for users at all.